flytekit/remote/remote.py

```diff
@@ -2101,7 +2101,7 @@ def sync_node_execution(
                 sub_node_mapping = {n.id: n for n in sub_flyte_workflow.flyte_nodes}
                 execution._underlying_node_executions = [
                     self.sync_node_execution(FlyteNodeExecution.promote_from_model(cne), sub_node_mapping)
-                    for cne in child_node_executions
+                    for cne in child_node_executiFsync_execons
```

Comment:
where is `child_node_executiFsync_execons` coming from? also kind of a strange-looking variable name...

Reply:
uh, not sure how that happened, thanks